File: Data Structures & Algorithms/find-target-in-rotated-sorted-array/submission-0.py
class Solution:
    def search(self, nums: List[int], target: int) -> int:
        # input: List: nums , int : target
        #output: int:  index of target or -1

        # brute force 
        for index, value in enumerate(nums):
            if value == target:
                return index

        return -1

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        //brute force

        for(int i = 0 ;i<nums.length-1;i++){
            for(int j = 0;j<nums.length;j++){
                if(nums[i]+ nums[j]== target) return new int[]{i+1,j+1};
            }

        }
        return new int[]{0,0};
    }
}
